examples/seismic/utils.py

```diff
@@ -62,6 +62,10 @@ def __init__(self, model, rec_positions, src_positions, t0, tn, **kwargs):
         In practice would be __init__(segyfile) and all below parameters
         would come from a segy_read (at property call rather than at init)
         """
+        if src_positions.ndim == 1:
```

Comment:
`if` part probably not necessary, `src_positions = np.reshape(src_positions, (-1, model.dim))` will work either way